src/Favourites/model.js

```diff
@@ -5,14 +5,15 @@ const connection = require("../db/connection");
 const Favourite = connection.define(
     "Favourite",
     {
-        favourite: {
```

Comment:
Is this not needed anymore? How will the user select as their favourite now?

Reply:
this has been changed